.github/workflows/image-push-master.yml

```diff
@@ -1,89 +1,128 @@
 name: "Push images on merge to master"
+
+env:
+  IMAGE_NAME: ghcr.io/${{ github.repository }}
+  BUILD_PLATFORMS: linux/amd64,linux/arm64,linux/ppc64le
+
 on:
   push:
     branches:
       - master
+
 jobs:
   build-and-push-image-operator:
     runs-on: ubuntu-20.04
-    env:
-      IMAGE_NAME: ghcr.io/${{ github.repository }}
     steps:
       - name: Check out the repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
+      # Add support for more platforms with QEMU (optional)
+      # https://github.com/docker/setup-qemu-action
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
 
       - name: Login to Docker
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Docker meta
+        id: docker_meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.IMAGE_NAME }}
+
       - name: Build and push sriov-network-operator
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v4
         with:
           context: .
           push: true
+          platforms: ${{ env.BUILD_PLATFORMS }}
           tags: |
             ${{ env.IMAGE_NAME }}:latest
```

Comment:
this wont mess up the tags ?

we want to have a latest tag per platform, then create and push a manifest which will be with "latest" lag containing platform specific images.